### PR TITLE
[New] Add jSignature@2.1.2 w/ git auto-update

### PR DESCRIPTION
**Pull request by finch185277**
Pull request for issue: #11455 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/brinley/jSignature/
 * Official website (optional, not the repository): http://www.unbolt.net/jSignature
 * NPM package url (optional):https://www.npmjs.com/package/jsignature
 * License and its reference:MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers:38
   - Count of stars:299
   - Count of forks:310
 * NPM download stats (optional):
   - Downloads in the last day:7
   - Downloads in the last week:52
   - Downloads in the last month:151

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jsignature/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "author": "Brinley Ang",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/brinley/jSignature.git",
+    "fileMap": [
+      {
+        "basePath": "libs",
+        "files": [
+          "**/*"
```
Comment (sashberd) [inline ajax/libs/jsignature/package.json]: Have a[ look here](https://github.com/brinley/jSignature/tree/v2.1.2#adding-jsignature-to-your-page) which files were used bu author and update the pattern accordingly

File: ajax/libs/jsignature/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/brinley/jSignature"
+  },
+  "keywords": [
+    "jSignature",
+    "signature",
+    "jQuery"
+  ],
+  "author": "Brinley Ang",
```
Comment (sashberd) [inline ajax/libs/jsignature/package.json]: Please add homepage property with http://www.unbolt.net/jSignature value

**Comment by finch185277** [comment]
I add all items in libs folder

"basePath": "libs",
"files": [
 "**/*"

because the author said "Look into libs folder in the source tree. Pick 3 files from there: ..."

maybe I am wrong.

File: ajax/libs/jsignature/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jsignature",
+  "version": "2.1.2",
+  "filename": "jSignature.min.js",
```
Comment (sashberd) [inline ajax/libs/jsignature/package.json]: Lets (always) set filename property after the name property

File: ajax/libs/jsignature/package.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "jsignature",
```
Comment (extend1994) [inline ajax/libs/jsignature/package.json]: Let's update this to `jSignature` which is exactly the same as the repo name, also don't forget to update folder name.

File: ajax/libs/jsignature/package.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/brinley/jSignature.git",
+    "fileMap": [
+      {
+        "basePath": "libs",
+        "files": [
+          "jSignature*",
+          "flashcanvas*"
```
Comment (extend1994) [inline ajax/libs/jsignature/package.json]: Let's combine the rules to `+(jSignature|flashcanvas)*`.

**Pull request merged**